go-platform

```diff
@@ -18,6 +18,7 @@ case $(uname -m) in
   x86_64) GOARCH=amd64 ;;
   i686) GOARCH=386 ;;
   i686) GOARCH=386 ;;
+  arm) GOARCH=arm GOARM=7 ;;
```

Comment:
does `GOARM` need to be printed too? `go-platform` right now only prints `$GOOS-$GOARCH`.